src/containers/shared/components/Transaction/PaymentChannelCreate/Simple.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { DATE_OPTIONS } from '../../../transactionUtils'
+import { localizeNumber, localizeDate } from '../../../utils'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+import { SimpleRow } from '../SimpleRow'
+import { useLanguage } from '../../../hooks'
+import { TransactionSimpleProps } from '../types'
+import { PaymentChannelCreateInstructions } from './types'
+
+export const Simple = ({
+  data,
+}: TransactionSimpleProps<PaymentChannelCreateInstructions>) => {
+  const language = useLanguage()
+  const { t } = useTranslation()
+  const { amount, source, destination, delay, cancelAfter, channel } =
+    data.instructions
+  const caDate = localizeDate(Date.parse(cancelAfter), language, DATE_OPTIONS)
```

Comment:
nit: This is only used twice, so I'd go for the more descriptive name 
```suggestion
  const cancelAfterDate = localizeDate(Date.parse(cancelAfter), language, DATE_OPTIONS)
```